db/db_options_test.cc

```diff
@@ -316,6 +316,40 @@ TEST_F(DBOptionsTest, MaxTotalWalSizeChange) {
   }
 }
 
+TEST_F(DBOptionsTest, DeleteObsoleteFilesPeriodChange) {
+  Options options;
```

Comment:
You can set a `SpecialEnv` (see db/db_test_util.h) to options to mock `Env->NowMicros()`.